pkg/tiller/release_update_test.go

```diff
@@ -23,12 +23,13 @@ import (
 
 	"github.com/golang/protobuf/proto"
 
+	"reflect"
```

Comment:
this needs to be moved up one more level (with the rest of the stdlibs). Thanks for catching this!

Reply:
Done.